rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -44,6 +44,12 @@ THE SOFTWARE.
 #endif
 #include "randombboxcrop_meta_data_reader.h"
 #define MAX_STRING_LENGTH 100
+#define MAX_OBJECTS 50 // Max number of objects/image in COCO dataset is 93 
+#define BBOX_COUNT 4
+#define MAX_NUM_ANCHORS 8732
```

Comment:
Please add comments on why we are setting max anchors here.

Reply:
@fiona-gladwin Please add this